Rakefile

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 $white_label_dir = 'codegen/smithy-ruby-codegen-test/build/smithyprojections/smithy-ruby-codegen-test/white-label/ruby-codegen/white_label'
+$rpcv2_cbor_dir = 'codegen/smithy-ruby-codegen-test/build/smithyprojections/smithy-ruby-codegen-test/protocoltests-rpcv2cbor/ruby-codegen/rpcv2_cbor'
```

Comment:
nit: Probably can just name this service rpcv2cbor similar to our rails json which is also protocol tests.

Reply:
I went with `rpcv2_cbor` to match `rails_json` (and the protocol shape is `Rpcv2Cbor` so `rpcv2_cbor` is I think the natural snake case of that).

Reply:
I was talking about `protocoltests-rpcv2cbor`

Reply:
oh, I see.  updated.